Cargo.toml

```diff
@@ -119,7 +119,8 @@ harness = false
 members = [
     "pyo3-macros",
     "pyo3-macros-backend",
-    "examples"
+    "examples",
+    "xtask"
```

Comment:
We don't necessarily need to name this "xtask". I don't have a motivation for changing it. But we could.

Reply:
Yeah, I don't personally _love_ the name xtask but I do like that it's a documented "approach" (https://github.com/matklad/cargo-xtask), so without good motivation I don't feel a need to change it.

For example I'd personally enjoy the alias being called `cargo run-script`. There's nothing stopping us adding a couple extra aliases that all do the same later, if we wanted ;)